packages/nuxt/src/vite/sourceMaps.ts

```diff
@@ -27,21 +33,25 @@ export function setupSourceMaps(moduleOptions: SentryNuxtModuleOptions, nuxt: Nu
 
   nuxt.hook('nitro:config', (nitroConfig: NitroConfig) => {
     if (sourceMapsEnabled && !nitroConfig.dev) {
-      if (nitroConfig.rollupConfig) {
-        // Add Sentry plugin
-        if (!Array.isArray(nitroConfig.rollupConfig.plugins)) {
-          nitroConfig.rollupConfig.plugins = nitroConfig.rollupConfig.plugins ? [nitroConfig.rollupConfig.plugins] : [];
-        }
+      if (!nitroConfig.rollupConfig) {
+        nitroConfig.rollupConfig = {};
+      }
+
+      if (isNullValue(nitroConfig.rollupConfig.plugins)) {
```

Comment:
l: Personal preference, I would just do `!== null` etc instead of the helper because it is easier to parse mentally but I'll leave it up to you.